integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/automaticindexing/AutomaticIndexingOutOfTransactionIT.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.mapper.orm.automaticindexing;
+
+import static org.hibernate.search.util.impl.integrationtest.orm.OrmUtils.withinSession;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+import org.hibernate.FlushMode;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.AvailableSettings;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.GenericField;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+import org.hibernate.search.util.impl.integrationtest.common.rule.BackendMock;
+import org.hibernate.search.util.impl.integrationtest.orm.OrmSetupHelper;
+import org.hibernate.search.util.impl.test.annotation.PortedFromSearch5;
+import org.hibernate.search.util.impl.test.annotation.TestForIssue;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+@TestForIssue(jiraKey = "HSEARCH-3068")
+public class AutomaticIndexingOutOfTransactionIT {
+
+	@Rule
+	public BackendMock backendMock = new BackendMock( "myBackend" );
+
+	@Rule
+	public OrmSetupHelper ormSetupHelper = OrmSetupHelper.withBackendMocks( backendMock );
+
+	private SessionFactory sessionFactory;
+
+	@Before
+	public void before() {
+		backendMock.expectAnySchema( IndexedEntity.INDEX_NAME );
+		sessionFactory = ormSetupHelper
+				.start()
+				.withProperty( AvailableSettings.ALLOW_UPDATE_OUTSIDE_TRANSACTION, true )
+				.setup( IndexedEntity.class );
+		backendMock.verifyExpectationsMet();
+	}
+
+	@Test
+	@PortedFromSearch5(original = "TransactionTest#testTransactionCommit")
```

Comment:
This is not at all the same test as `TransactionTest#testTransactionCommit`, though, since the original was inside a transaction and this one is not.

Also, if you add `@PortedFromSearch5` annotations, please use the fully qualified name of the class, and mark the original method with `@Category(PortedToSearch6.class)`. This will help later when we'll try to port more tests from Search 5 to 6.

Reply:
You're right tests are not equals. So I removed the annotation from there.